modules/ossm-customize-smmr-cluster-wide.adoc

```diff
@@ -0,0 +1,26 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
+
+:_content-type: CONCEPT
+[id="ossm-customize-smrr-cluster-wide_{context}"]
+= Customizing the member roll for cluster-wide mesh
+
+In cluster-wide mode, when you create the `ServiceMeshControlPlane` resource, the `ServiceMeshMemberRoll` resource is also created. You can modify the `ServiceMeshMemberRoll` resource after it gets created. Once you modify the resource, the {SMProductShortName} operator no longer changes it. If you modify the `ServiceMeshMemberRoll` resource using the {product-title} web console accept the prompt that indicates the modifications may be overwritten.
```

Comment:
```suggestion
In cluster-wide mode, when you create the `ServiceMeshControlPlane` resource, the `ServiceMeshMemberRoll` resource is also created. You can modify the `ServiceMeshMemberRoll` resource after it gets created. After you modify the resource, the {SMProductShortName} operator no longer changes it. If you modify the `ServiceMeshMemberRoll` resource by using the {product-title} web console, accept the prompt to overwrite the modifications.
```
- Per ISG, only use "once" to indicate "one time" and not as a synonym for "after": https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences
- Add commas for clarity
- Per ISG, don't use "may" as a synonym for "might" or "can".

Reply:
Done